### PR TITLE
[Feature] Remove download implemented in Dashboard

### PR DESCRIPTION
**Pull request by saisankargochhayat**
Solves #102.
Added a delete button in the dashboard.
image
Currently it works if the download id is less than 10.
If the id is 10 it fails . 
@malithsen if you could give a look.

### CODE REVIEW

File: ui/src/app/views/dashboard.html [hunk 1]
```diff
@@ -16,14 +16,18 @@ <h4 class="panel-widget-tittle"> On going downloads</h4>
     <table id="table" class="table table-hover table-bordered">
         <thead>
         <tr ng-repeat="data in downloads">
-            <td data-title="ID">{{$index + 1}}</td>
-            <td data-title="Issue">{{data.download_name}}</td>
-            <td data-title="Progress">
+            <td style="padding-top: 3%" data-title="ID">{{$index + 1}}</td>
```
Comment (malithsen) [inline ui/src/app/views/dashboard.html]: Let's not use inline styles

File: components/core/DownloadManager.py [hunk 2]
```diff
@@ -36,15 +36,15 @@ def remove_download(id, userName):
     db = threadpool.connect()
     if db is not None:
         cursor = db.cursor()
-        sql1 = "SELECT status FROM download WHERE id=%s;"
+        sql1 ="SELECT status FROM download WHERE id=%s;"
         sql = "DELETE from download WHERE id=%s and user_name=%s;"
         try:
-            cursor.execute(sql1, id)
+            cursor.execute(sql1,str(id))
```
Comment (saisankargochhayat) [inline components/core/DownloadManager.py]: If the value of id exceeds 10 , delete fails. Please suggest something regarding this.

File: ui/src/app/views/dashboard.html (hunk 1 above)
Comment (saisankargochhayat) [inline ui/src/app/views/dashboard.html]: Will remove them :)

**Comment by saisankargochhayat** [comment]
The issue #102 has been fixed . Currently works for all id values . Please review :)

File: ui/src/app/stylesheets/_table.scss [hunk 3]
```diff
@@ -210,3 +210,7 @@ $table-border-color:            #e0e0e0;
     }
   }
 }
+
+.vertalign{
+  padding-top: 3% !important;
```
Comment (malithsen) [inline ui/src/app/stylesheets/_table.scss]: Make it an id and you can get rid of the `!important`. Also IMO _custom.scss is a better place to have this.

**Comment by saisankargochhayat** [comment]
Added a confirmation dialogue . :smile: @malithsen
pr

**Pull request merged**